app/lib/actions/katello/repository_set/disable_repository.rb

```diff
@@ -18,8 +18,8 @@ def humanized_name
           _("Disable")
         end
 
-        def plan(product, content, substitutions)
-          if repository = repository_mapper(product, content, substitutions).find_repository
+        def plan(product, content, substitutions, registry_name = nil)
```

Comment:
@jlsherrill  Are you ok with the approach though. Also do you just want me to change this in that 1 plan file or do you want me to rewrite RepositoryMapper and DockerRepository mapper to accept substitutions and make em mean different things in each one ?